python/gui/MaaS/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Put a module wide description here
```

Comment:
TODO?

Reply:
Pretty much. I'm probably going to go ahead and do a find and replace on my current work.